ledger/participant-integration-api/src/main/resources/db/migration/postgres-appendonly/V119__add_transaction_metering.sql

```diff
@@ -0,0 +1,18 @@
+--  Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+--  SPDX-License-Identifier: Apache-2.0
+
+SELECT 'Add Transaction Metering: Creating table...';
+
+CREATE TABLE transaction_metering (
+    application_id VARCHAR(512) NOT NULL,
```

Comment:
`application_id NVARCHAR2(1000) NOT NULL,`
vs 
`application_id VARCHAR(512) NOT NULL,`
Why different sizes?

Reply:
@meiersi-da recommended 512.